posts/peachpellen/5_Apps_for_Android.jade

```diff
@@ -0,0 +1,56 @@
+---
+title:      '5 Apps for Android Developers'
+categories: [ideate, launch, design, build]
+heroimage:  ''
```

Comment:
Looks like you need to set a heroimage: please add an image to the library folder created for you and reference it here. The template will pick it up and use it in the header of your post and any listing pages.

Thanks!